services/auth.service.js

```diff
@@ -84,31 +90,27 @@ async function ensureAuthorized(req, findByIdFns) {
     for (const route of routes) {
         // if the request types are different, go to next
         // the incoming request type is in req.method according to node v10.12.0 api
-        if (route.requestType !== req.method)  {
+        if (route.requestType !== req.method) {
             continue;
         }
 
-        let splitRoute = route.uri.split("/");
+        let routeUri = route.uri;
+        if (routeUri.slice(-1) !== "/") {
```

Comment:
Same with comment for this, just so that in the future we actually know what's going on

Reply:
Done